packages/manager/apps/pci-gateway/tailwind.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   content: [
     './src/**/*.{js,jsx,ts,tsx}',
     '../../../manager-react-components/src/**/*.{js,jsx,ts,tsx}',
+    '../../modules/manager-pci-common/**/*.{js,jsx,ts,tsx}',
```

Comment:
@fredericvilcot  are we sure about that ?

Reply:
No sorry 👍  Its seems to be OK

Reply:
@aboungnaseng-ovhcloud 
yes, it's mandatory to import tailwind style from `manager-pic-common`, if not UI is broken.